### PR TITLE
[BUGFIX] make sure currencies param of requestAccount is string array

### PR DESCRIPTION
**Pull request by chabroA**
### 📝 Description

If the currencies array passed to `requestAccount` contains other values than string, the underlying function crashes (cf. demo). 

Make sure currencies param of requestAccount is a string array.

### ❓ Context

- **Impacted projects**: `` 
- **Linked resource(s)**: `` 

### ✅ Checklist

- [ ] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 📸 Demo

<details>
<summary>KO 💥 </summary>

https://user-images.githubusercontent.com/9203826/208966317-7a0a5d13-11ee-4e9c-bf53-67a099a5e05a.mov

</details>

<details>
<summary>OK ✅  </summary>

https://user-images.githubusercontent.com/9203826/208966376-6bfbf50d-d7f8-4f2d-b121-207e6c091f85.mov

</details>

### 🚀 Expectations to reach

LL should not crash even if invalid values passed as param for live-app-sdk functions

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: .changeset/five-dragons-jog.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+"ledger-live-desktop": patch
+"live-mobile": patch
```
Comment (Justkant) [inline .changeset/five-dragons-jog.md:3]: You'll need to split this changeset

**Review by Justkant — APPROVED** [review]
IIUC it shouldn't be an issue with the wallet-api because we use zod to validate at runtime right ?

**Comment by Justkant** [comment]
You have a lint error with prettier, just need a quick format on the webview file

**Comment by chabroA** [comment]
> IIUC it shouldn't be an issue with the wallet-api because we use zod to validate at runtime right ?

Indeed it should not. In this case it should throw an error on the wallet-api-client side at runtime (so at worst it's the live app that will crash if they don't handle errors for example) 
Wallets should be safe 👍

**Pull request merged**